security-center/snippets/system-test/v2/notifications.test.js

```diff
@@ -27,7 +27,7 @@ const exec = cmd => execSync(cmd, {encoding: 'utf8'});
 
 // TODO(developers): update for your own environment
 const organizationId = '1081635000895';
-const projectId = 'long-door-651';
+const projectId = 'project-a-id';
```

Comment:
question: Shouldn't these be populated from environment variables configured on the workflow?

Reply:
Addressed.